scripts/importmcdev.sh

```diff
@@ -69,6 +69,7 @@ for f in $files; do
 	fi
 done
 
+
```

Comment:
?

Reply:
it wasnt me this time :P

Reply:
i merged branch and had to solve conflict, and githubs conflict resolver doesnt make it clear the diff.